packages/client/src/LedgerLivePlatformSDK.ts

```diff
@@ -261,7 +261,15 @@ export default class LedgerLivePlatformSDK {
    */
   async listAccounts(
     params: {
+      /**
+       * Include tokens accounts in the results
+       */
       includeTokens?: boolean;
```

Comment:
@Justkant if I remember well, we wanted to remove the whole concept of `includeTokens` ?